packages/muelu/test/meshtying/meshtying2D_medium_simple_spai.xml

```diff
@@ -0,0 +1,229 @@
+      <Parameter name="Nullspace" type="string" value="myNspFact1"/>
+      <Parameter name="CoarseMap" type="string" value="myCoarseMap1"/>
+    </ParameterList>
+
+    <!-- Multigrid setup for dual block (A_{11}) -->
+    <ParameterList name="mySecondGroup">
+      <Parameter name="group" type="string" value="FactoryManager"/>
+      <Parameter name="use kokkos refactor" type="bool" value="false"/>
+      <Parameter name="A" type="string" value="mySubBlockAFactory2"/>
+      <Parameter name="P" type="string" value="myTentativePFact2"/>
+      <Parameter name="Aggregates" type="string" value="myInterfaceAggs2"/>
+      <Parameter name="Nullspace" type="string" value="myNspFact2"/>
+      <Parameter name="CoarseMap" type="string" value="myCoarseMap2"/>
+      <Parameter name="Primal interface DOF map" type="string" value="myInterfaceMapTransFact"/>
+    </ParameterList>
+
+    <!-- BLOCK TRANSFER operators -->
+
+    <!-- define block prolongation operator using above blocks -->
+    <ParameterList name="myBlockedPFact">
+      <Parameter name="factory" type="string" value="BlockedPFactory"/>
+      <!-- factory manager for block 1 -->
+      <ParameterList name="block1">
+         <Parameter name="group" type="string" value="myFirstGroup"/>
+      </ParameterList>
+      <!-- factory manager for block 2 -->
+      <ParameterList name="block2">
+        <Parameter name="group" type="string" value="mySecondGroup"/>
+      </ParameterList>
+    </ParameterList>
+
+    <!-- define block restriction operator using above blocks -->
+    <ParameterList name="myBlockedRFact">
+      <Parameter name="factory" type="string" value="GenericRFactory"/>
+      <Parameter name="P" type="string" value="myBlockedPFact"/>
+    </ParameterList>
+
+    <ParameterList name="myBlockedRAPFact">
+      <Parameter name="factory" type="string" value="BlockedRAPFactory"/>
+      <Parameter name="P" type="string" value="myBlockedPFact"/>
+      <Parameter name="R" type="string" value="myBlockedRFact"/>
+    </ParameterList>
+
+    <!-- BLOCK SMOOTHERS -->
+    <ParameterList name="mySmooFact1">
+      <Parameter name="factory" type="string" value="TrilinosSmoother"/>
+      <Parameter name="type" type="string" value="RELAXATION"/>
+      <ParameterList name="ParameterList">
+        <Parameter name="relaxation: type" type="string" value="Symmetric Gauss-Seidel"/>
+        <Parameter name="relaxation: sweeps" type="int" value="1"/>
+        <Parameter name="relaxation: damping factor" type="double" value="0.4"/>
+      </ParameterList>
+    </ParameterList>
+
+    <!-- Build Schur Complement factory (for being used in SIMPLE type block smoother)-->
+    <ParameterList name="myInvApproxFact">
+      <Parameter name="factory" type="string" value="InverseApproximationFactory"/>
+      <Parameter name="A" type="string" value="mySubBlockAFactory1"/>
+      <Parameter name="inverse: approximation type" type="string" value="spai"/>
```

Comment:
Similar to my other comment, please spell out the "spai" abbreviation, otherwise it's really hard to understand.